atmos_model.F90

```diff
@@ -840,8 +826,8 @@ subroutine update_atmos_model_state (Atmos)
     call atmosphere_get_bottom_layer (Atm_block, DYCORE_Data)
 
     !if in coupled mode, set up coupled fields
```

Comment:
This means for all runs, including `APP=ATM` atmosphere only runs, just not for coupled chemistry runs? Should the comments in lines 828 and 268 be updated?